docs/src/solutions.md

```diff
@@ -52,6 +52,12 @@ This function will return a `MOI.TerminationStatusCode` `enum`.
 MOI.TerminationStatusCode
 ```
 
+Additionally, we can receive a solver specific string explaning why the
+optimization stopped with
+```julia
+raw_status(model)
+```
```

Comment:
It's better to go 
```
optimization stopped with [`raw_status`](@ref).
```

and then add `raw_status` to the reference list at the bottom.